content/explore-all-resources/staff-wellbeing.html.erb

```diff
@@ -9,90 +9,118 @@ layout: /explore_resources.html.erb
 
 <div class="explore-resource-card-group">
   <%= render('/partials/explore_resource_card.*',
-    title: "Establish a wellbeing committee",
-    href: "#{@base_url}/staff-wellbeing/establish-a-wellbeing-committee",
+    title: "Create a school culture which improves wellbeing",
+    href: "#{@base_url}/staff-wellbeing/create-a-school-culture-which-improves-wellbeing",
     tag:  "Wellbeing",
     details: {
-      resource_type: "Example",
-      reading_time: "3 minutes",
-      created_by: "Notre Dame High School"
+      resource_type: "Case study",
+      reading_time: "4 minutes",
+      created_by: "Kensington Primary School"
     }
   )%>
 
+<div class="explore-resource-card-group">
```

Comment:
You've got an extra div with this "explore-resource-card-group" class at the start of each render block.

They need to be removed, except for the original one on line 10.

```suggestion
```
This means the explore page now looks like this!

<img width="347" alt="Screenshot 2024-03-01 at 11 01 00" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/dc1034fc-be90-45d8-b859-c1913963c342">

Reply:
I think I've fixed this, thank you